lib/editor.coffee

```diff
@@ -32,6 +32,11 @@ textEditor = ($item, item, option={}) ->
 
   keydownHandler = (e) ->
 
+    if e.which == 27 #esc for save
+      e.preventDefault()
+      $textarea.focusout()
+      return false
+
     if (e.altKey || e.ctlKey || e.metaKey) and e.which == 83 #alt-s for save
```

Comment:
I am minded to change this to be only `e.ctlKey || e.metaKey)` as including alt-s prevent ß from being typed *at least on a UK Apple keyboard*.

Reply:
also tempted to remove the other use of alt combination - alt-i and alt-m used for circumflex diacritic and 'µ'.

Reply:
This was addressed in https://github.com/fedwiki/wiki-client/pull/240